entrypoint.sh

```diff
@@ -392,6 +401,24 @@ then
             helm cm-push --force ".tmp/charts/${chart_name}-${INPUT_CHART_VERSION}.tgz" repo
             ;;
 
+        helm-publish-oci)
+            is_set INPUT_CHART_APP_VERSION
+            is_set INPUT_CHART_PATH
+            is_set INPUT_CHART_VERSION
+            is_set INPUT_CHART_REPO
+            is_set INPUT_CHART_REPO_PASSWORD
+            is_set INPUT_CHART_REPO_USERNAME
+
+            helm_package
+
+            echo "-- Login OCI registry ${INPUT_CHART_REPO}, as ${INPUT_CHART_REPO_USERNAME}"
+            echo "${INPUT_CHART_REPO_PASSWORD}" | helm registry login ${INPUT_CHART_REPO} --username ${INPUT_CHART_REPO_USERNAME} --password-stdin
```

Comment:
Shellcheck: Wrap variables in quotes.

Reply:
```
In entrypoint.sh line 415:
            echo "${INPUT_CHART_REPO_PASSWORD}" | helm registry login ${INPUT_CHART_REPO} --username ${INPUT_CHART_REPO_USERNAME} --password-stdin
                                                                      ^-----------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                                                     ^--------------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```